src/condor_schedd.V6/qmgmt.cpp

```diff
@@ -6416,6 +6417,18 @@ int UpdateUserRecAttributes(JobQueueKey & key, bool is_project, const ClassAd &
 			continue;
 		}
 
+		// treat Attr = delete or Attr = <nullptr> as a request to delete the attribute
+		// and not a request to set the attribute
+		if ( ! tree || (ExprTreeIsAttrRef(SkipExprParens(tree), delref) && delref=="delete")) {
```

Comment:
Is this case insensitive? Do we support case insensitive deletion (i.e. user specifies `Delete`)

Reply:
I debated case-sensitivity here, decided to make it sensitive for now.